Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -270,6 +270,18 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
         BackgroundContentFailedToLoad
     };
 
+    enum class PermissionNotification : uint8_t {
+        Unknown = 0,
```

Comment:
`None` seems better for this.

```suggestion
        None = 0,
```